src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -34,7 +34,8 @@
 public class ParserUtil {
 
     public static final String MESSAGE_INVALID_INDEX = "Index is not a non-zero unsigned integer.";
-    private static final String DURATION_HOURS_REGEX = "^[0-9]*H";
+
+    private static final String DURATION_HOURS_REGEX = "^[0-9]*H?";
```

Comment:
Is there suppose to be a '?' here? Just a bit suspicious since this caused us some problems before.

Reply:
Yup, the ? indicates that the H is optional

Reply:
Yup, the ? means that H itself is optional. So if a user does not specify H or M we just take it as hours.

Reply:
Oh I see